fava/static/javascript/charts.js

```diff
@@ -539,9 +539,14 @@ class LineChart extends BaseChart {
       min(this.data, s => s.values[0].date),
       max(this.data, s => s.values[s.values.length - 1].date),
     ]);
+
+    // Span y-axis min to max value plus margin as percent of either value
+    this.yMarginPercent = 0.03;
```

Comment:
Don't think we need a variable for this, just use it inline

Reply:
My argument in favor of it is that the value is used more than once and involves a "magic value". I am happy to inline it if you still feel it should go that route.

Reply:
It also saves an extra calculation haha

Reply:
After playing with the values a bit, 5% looks better. Code also looks nicer with the value inline. I'd advocate for the variable more if it was used somewhere outside the local scope, as it is, it seems clear enough.